ndcube/ndcollection.py

```diff
@@ -222,8 +222,12 @@ def _generate_collection_getitems(self, item):
         return collection_items, new_aligned_axes
 
     def copy(self):
-        return self.__class__(self.items(), tuple(self.aligned_axes.values()),
-                              meta=self.meta, sanitize_inputs=False)
+        # Aligned axes is not a required parameter and may be None
+        if self.aligned_axes is not None:
+            return self.__class__(self.items(), tuple(self.aligned_axes.values()),
+                                  meta=self.meta, sanitize_inputs=False)
+        else:
+            return self.__class__(self.items(), None, meta=self.meta, sanitize_inputs=False)
```

Comment:
The logic looks correct here.  I think it could be written a little more succinctly.  See below.
```suggestion
        aligned_axes = None if self.aligned_axes is None else tuple(self.aligned_axes.values())
        return self.__class__(self.items(), aligned_axes, meta=self.meta, sanitize_inputs=False)
```

Reply:
Funny enough, I thought about doing something like this from the start! I was weighing whether I liked the conciseness of the ternary operator of the more verbose clarity of the conditional block.